### PR TITLE
Fix responsive issues in userstore connection detail update page

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject

- On large screens

Screenshot 2023-10-17 at 10 21 21

- On macbook screen

Screenshot 2023-10-17 at 10 22 03

### Related Issues
- https://github.com/wso2/product-is/issues/16950

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx [hunk 1]
```diff
@@ -309,47 +309,40 @@ export const EditConnectionDetails: FunctionComponent<EditConnectionDetailsProps
...
-                                                                {
-                                                                    name: property.description.split("#")[ 0 ]
-                                                                })
-                                                            }
-                                                            data-testid={ 
-                                                                `${ testId }-form-password-input-${ property.name }` 
-                                                            }
+                                                    <Box display="flex" flexDirection="row">
+                                                        <Checkbox 
+                                                            sx={ { pb: 1.4, pl: 0,pr: 1, pt: 0 } }
```
Comment (Achintha444) [inline apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx:314]: Is it okay to use sx prop ?

Comment (pavinduLakshan) [reply apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx:314]: It'd be okay I think. It has been already used in the codebase.

Comment (brionmario) [reply apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx:314]: Ideally, we should have a component-specific `.scss` if this is not meant to be reused.

Ref: https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/authentication-flow-builder/components/predefined-flows-side-panel/predefined-flows-side-panel.scss

Comment (pavinduLakshan) [reply apps/console/src/features/userstores/components/edit/edit-connection-details-user-store.tsx:314]: Will fix this in a follow-up PR

**Pull request merged**